.vib/redis/goss/redis.yaml

```diff
@@ -0,0 +1,15 @@
+file:
+  /opt/bitnami/redis/etc/redis.conf:
+    exists: true
+    mode: "0664"
+    contains:
+      - port {{ .Vars.port }}
+      - dir /bitnami/redis/data
+      - pidfile /opt/bitnami/redis/tmp/redis.pid
+      - daemonize yes
+command:
+  check-redis-server:
+    exec: timeout 5s redis-server || true
```

Comment:
Why are you using `|| true` here?

Reply:
Timeout does not exit with status 0. Using || true, seems cleaner.

Reply:
If we are configuring Redis to be used as a daemon, couldn't we initialize it so using the `redis.conf` file?

Reply:
I've changed it and it's better, as it exits properly, and then we can check if the process is running.